debezium-api-generator/README.md

```diff
@@ -0,0 +1,48 @@
+# Debezium OpenAPI Generator
```

Comment:
README needs some more love.

Reply:
No, I think main repo is good. Any cross-repo change always is a hassle, so let's try and minimize the need for this.

Reply:
The number of cross-repo changes are not of a concern here, the Metadata classes are in the connectors. Quite strict SOC.